Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage.yml

```diff
@@ -35,8 +35,7 @@ configuration:
     defaultvalue: 3 days
     type: 0
     required: false
-  - display: Category ID to fetch - corresponds to the ranking that determines the Cofense
-      Triage prioritization (1-5)
+  - display: 'Category ID to fetch - corresponds to category IDs defined in Triage'
```

Comment:
As we discussed in the call, let's add an explanation of the possible numbers, and if only a single value or range is available.
also, from version 5.0.0, which is what we will do in the v2, you cand add a tooltip to any integration parameter you want using the key `additionalinfo`